__tests__/src/utils/chat.js

```diff
@@ -0,0 +1,25 @@
+import createGiftedChatMessageObject from '../../../src/utils/chat';
```

Comment:
Please fix ESLint issues for that file.

Reply:
Does this file has ESLint issue?

Reply:
Yes, please run `./node_modules/.bin/eslint __tests__/src/utils/chat.js` in terminal from root project folder to check.

Reply:
You can also use `./node_modules/.bin/eslint __tests__/src/utils/chat.js --fix` to apply automatic fixes.